submissions/__init__.py

```diff
@@ -1,2 +1,2 @@
 """ API for creating submissions and scores. """
-__version__ = '3.1.6'
+__version__ = '3.1.7'
```

Comment:
Does it make sense to increment version if we are just adding doc string/fixing unit tests?

Reply:
I think so, my understanding is that it goes <major>.<minor>.<build> so really any code change should bump that last number